test/e2e/testdata/app-exclusions/deploymentDoc.yaml

```diff
@@ -0,0 +1,16 @@
+##Purpose of this file is to test this kind of files should be ignored from marshalling since it is lacking of apiVersion, kind, and metadata
```

Comment:
Is this somehow being used in a e2e test?

Reply:
Yes. This is used in this test https://github.com/argoproj/argo-cd/blob/master/test/e2e/app_management_test.go#L1407